cmd/operator-sdk/build/cmd.go

```diff
@@ -172,15 +176,31 @@ func buildFunc(cmd *cobra.Command, args []string) error {
 
 	log.Infof("Building Docker image %s", baseImageName)
 
-	dbArgs := []string{"build", ".", "-f", "build/Dockerfile", "-t", baseImageName}
+	var buildCmd *exec.Cmd
+	switch imageBuilder {
+	case "docker":
+		dbArgs := []string{"build", ".", "-f", "build/Dockerfile", "-t", baseImageName}
 
-	if dockerBuildArgs != "" {
-		splitArgs := strings.Fields(dockerBuildArgs)
-		dbArgs = append(dbArgs, splitArgs...)
+		if dockerBuildArgs != "" {
+			splitArgs := strings.Fields(dockerBuildArgs)
+			dbArgs = append(dbArgs, splitArgs...)
+		}
+
+		buildCmd = exec.Command("docker", dbArgs...)
+	case "buildah":
+		bbArgs := []string{"bud", "--format=docker", "-f", "build/Dockerfile", "-t", baseImageName, "."}
+
+		if buildahBuildArgs != "" {
+			splitArgs := strings.Fields(buildahBuildArgs)
+			bbArgs = append(bbArgs, splitArgs...)
```

Comment:
You'll need to append `buildahBuildArgs` before `.`, as `buildah` doesn't allow args to come after build context.